explore/circadian-efficacy/data_id_mapping.ipynb

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
Let's always add a title markdown cell to our notebooks with a brief description of what they do.

Reply:
Also can you show the first two or three rows of each dataframe (`df.head(2)`) for all important dfs in the notebook. One of the best parts of notebooks is that they can show output interspersed with code. Without this, it is hard to understand the notebook.